internal/resources/diffValidation/resource_nsxt_lb_profile.go.go

```diff
@@ -0,0 +1,131 @@
+//  (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+package diffvalidation
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	serviceTypes    = "service_type"
+	httpProfile     = "http_profile"
+	tcpProfile      = "tcp_profile"
+	udpProfile      = "udp_profile"
+	cookieProfile   = "cookie_profile"
+	sourceIpProfile = "sourceip_profile"
```

Comment:
define these const in a separate file.

Reply:
let it be here because these const is not generic const to define in separate file